source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -382,6 +382,24 @@ def __init__(self):
             hidden=True,
             help="how long it takes to park the car, " "used especially in distributed scenario",
         )
+        parser_get.add_argument(
+            "_here_realtime_traffic",
+            type=six.text_type,
```

Comment:
The code could be much simpler if we use type=BooleanType() with default=True or False non?